testdata/src/go.uber.org/globalvars/globalvarinit.go

```diff
@@ -94,3 +96,14 @@ func foo() *int {
 	print(multiNonNil, multiNil, nonnilMethod, assignedNilable)
 	return nil
 }
+
+// Now we test a corner case where a constant is assigned to a global variable.
```

Comment:
```suggestion
// Below test checks when a constant is assigned to a global variable.
```

Reply:
Feel free to ignore this suggestion. 🙂  My reasoning was that the comment with "Now" makes sense in the context of this PR, but the story might be lost later in code.

Reply:
I agree, good catch! updated!